OKTA-1137444 OIN submission for 3p actions (#6095)
okta/okta-developer-docs: packages/@okta/vuepress-site/docs/guides/submit-oin-app/main/saml2/entry-path.md

2. On the **Home** page, do one of the following:

>     * Click **Create app** to create a new app that you want to list in the Okta Integration Network. <br><br>

Reviewer: ```suggestion
    * Click **Create app** to create an app that you want to list in the Okta Integration Network. <br><br>
```
